<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords" tests="2" time="0.000" ignored="0" failures="0" timestamp="07 Jun 2018 12:14:27 GMT" skipped="0" hostname="DESKTOP-QBEO8F3" errors="2">
  <testcase name="setUp" time="0.000" classname="com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords">
    <error type="java.io.FileNotFoundException" message="C:\Users\Progos-QA\workspace\PageObjectWithPageFactoryFramework-master\PageObjectWithPageFactoryFramework-master\src\main\java\com\test\automation\uiAutomation\config\config.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\Progos-QA\workspace\PageObjectWithPageFactoryFramework-master\PageObjectWithPageFactoryFramework-master\src\main\java\com\test\automation\uiAutomation\config\config.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.test.automation.practice.testBase.TestBase.loadData(TestBase.java:76)
	at com.test.automation.practice.testBase.TestBase.init(TestBase.java:86)
	at com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords.setUp(TC003_VerifyLoginWithDifferentRecords.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase name="verifyLoginWithDifferentRecords" time="0.000" classname="com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords">
    <error type="org.testng.TestNGException" message="
Data Provider public java.lang.String[][] com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords.getTestData() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.String;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.String[][] com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords.getTestData() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.String;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:133)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1020)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1110)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- verifyLoginWithDifferentRecords -->
</testsuite> <!-- com.test.automation.uiAutomation.homepage.TC003_VerifyLoginWithDifferentRecords -->
